### ISSUE TITLE
Kirin Tor patching

### ISSUE THREAD

**Issue by sebastienwood** [dalaranwow/dalaran-wow#4884]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Kirin Tor is far behind the improvements made to Algalon (quests, mobs, etc). It feels abandoned even though it has some pop on it and the progression feature not being here may appeal to some.

**Expected behaviour**: Roll patches to KT

**Steps to reproduce the problem**:

One major example include the quest chain for the Sons of Hodir, with [The Drakkensryd] quest (killing 10 drake riders before meeting Thorim) can't be completed. Many quests in Northrend doesn't work !

Another issue would be Wintergrasp, currently not active. I'd understand not activating it for pop. reasons, but then titanium nodes and greater elementals should be visible to everyone :)

**Include proofs for this behaviour**

**Comment by sebastienwood** [comment]
Up :)
Any thought from dev team on this issue at least ? Would it be feasible ? Would there be an interest to give Kirin Tor a new meaningful life ?

Thanks

**Comment by TeridanDW** [comment]
Duplicate with #4878 (Answer to your question is there).
